tests/baselines/reference/underscoreTest1.types

```diff
@@ -562,7 +562,7 @@ _.flatten([1, [2], [3, [[4]]]], true);
 >_.flatten : { <T>(list: T[][]): T[]; <T>(array: any[], shallow?: boolean): T[]; }
 >_ : Underscore.Static
 >flatten : { <T>(list: T[][]): T[]; <T>(array: any[], shallow?: boolean): T[]; }
->[1, [2], [3, [[4]]]] : (number | (number | number[][])[])[]
+>[1, [2], [3, [[4]]]] : (number | number[] | (number | number[][])[])[]

```

Comment:
This looks worse?  It has `number[]` twice.

Reply:
Right, but this is just a `.types` test. The real question is whether or not this ever gets observed elsewhere.